content/create-new-collection.en.md

```diff
@@ -0,0 +1,159 @@
+
+### Versions
+
+Create the snapshot repository by using the [Demo versions](https://github.com/OpenTermsArchive/demo-versions) as template:
+- Go to the [demo versions repository homepage](https://github.com/OpenTermsArchive/demo-versions)
+- Click on the "Use this template" dropdown
+- Select "Create a new repository"
+- In the opened page, fill the repository name, it is strongly recommended to use the `<collection_id>-versions`.
+- When redirected to the newly generated repositroy, please wait until the automatic setup is done (it should be done in less than a minute).
+- Give write rights on the repo to the maintenance team.
+
+#### Fill the about section
+
+- Set the description: “Documents versions for `<collection_name>`. Maintained by `<maintainer>`.”
+- Set website: https://docs.opentermsarchive.org/navigate-history/
+- Add the following tags: `terms-of-service`, `terms-of-service-agreements`, `terms-and-conditions`, `open-terms-archive`.
+- Uncheck “Packages” and “Environments”.
+
+#### Define repository settings
+
+These settings aim at minimising the otherwise overwhelming amount of information and click targets.
+
+- Features: uncheck “Wikis” and “Projects”.
+- In the `Actions` section, in the `General` submenu: select "Disable actions".
+
+## Set up the GitHub maintenance team
+
+For collections to be included in the Open Terms Archive organisation only. For third parties, handle rights however you see fit.
+
+- [Create a new team](https://github.com/orgs/OpenTermsArchive/new-team)
+- Give it the name of the collection
+- Set as avatar the collection icon from the website
+- Set as description: “Maintainers of the <collection_name> collection”
+- Add selected members to the team
+- Add the declarations repository to the collection team, with “Maintain” access rights
+- Add the snapshots repository to the collection team, with “Triage” access rights (giving them more would enable them to corrupt data)
+- Add the versions repository to the collection team, with “Triage” access rights (giving them more would enable them to corrupt data)
+- Add the declarations, snapshots and versions repositories to the Bots team with “Write” access
+
+- - -
+
+## Setup deployment
+
+### On the server
+
+- Connect to the server with `ssh <username>@<host>` (example usernames: `debian`, `ubuntu`…)
+- Create a new SSH key: `ssh-keygen -q -N "" -f ~/.ssh/ota-deploy`
+- Add public key to authorized_keys `cat ~/.ssh/ota-deploy.pub >> ~/.ssh/authorized_keys`
+	- Display the public key `cat ~/.ssh/ota-deploy.pub` and keep it temporarily for the next steps
+- Add private key to the SSH authentication agent `ssh-add ~/.ssh/ota-deploy` (start the ssh agent before if necessary `eval \`ssh-agent -s\``)
+	- Display the private key `cat ~/.ssh/ota-deploy` and keep it temporarily for the next steps
+
+Note: user must have the right to `sudo`.
+
+### On GitHub declarations repository settings
```

Comment:
```suggestion
### On GitHub definitions repository settings
```
It's really strange to have 2 different words for the same thing, we really need to correct this everywhere!

Reply:
I agree, and at this stage I believe it is safer to stick to “declarations”.